packages/synapse-react-client/test/components/SynapseTable/SynapseTableCell/JSON/__snapshots__/JSONTableCellRenderer.test.tsx.snap

```diff
@@ -229,7 +229,7 @@ exports[`JSONTableCellRenderer handles "Object of primitives" case 1`] = `
     </div>
   </div>
   <span
-    class="MuiTypography-root MuiTypography-smallText1 css-1i0dbvc-MuiTypography-root"
```

Comment:
So this class name was updated because there was a change to the theme?

Reply:
I think in this case it's because I changed the `sx` that was provided to the component.